node_modules/col-activities/lib/notifications/weekly.js

```diff
@@ -514,41 +522,75 @@ var UserTotals = function() {
     'pointsReceived': 0
   };
 
+  // Define a method to increment activity counts.
+  var activitiesDictionary = {
+    'get_asset_comment': 'commentsReceived',
+    'get_asset_comment_reply': 'commentsReceived',
+    'get_like': 'likesReceived'
+  };
+
+  that.incrementActivities = getActivitiesIncrementer(that, activitiesDictionary);
+
+  // Define a method to increment point totals.
+  var pointsDictionary = {
```

Comment:
@nicolaasmatthijs Actually it's not - users and courses increment different values - so predefining the dictionary here as suggested earlier is helpful.